data/scripts/talkactions/god/add_addon.lua

```diff
@@ -1,45 +1,305 @@
-local addons = TalkAction("/addaddon")
+local addaddon = TalkAction("/addaddon")
 
-function addons.onSay(player, words, param)
-	-- create log
-	logCommand(player, words, param)
+local looktypes = {
```

Comment:
This seems a bit redundant, as we already have the outfit list in the CPP file. Wouldn't it be better to just "get" it here, rather than hardcoding these values again? It would avoid having another place to update hardcoded IDs.